tests/ha/set_qos.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -xe
+
+NS_COUNT=400
+NQN="nqn.2016-06.io.spdk:cnode1QOS"
+GW1_NAME=$(docker ps --format '{{.ID}}\t{{.Names}}' | awk '$2 ~ /nvmeof/ && $2 ~ /1/ {print $1}')
+GW1_IP="$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' "$GW1_NAME")"
+cephnvmf="docker compose run --rm nvmeof-cli --server-address $GW1_IP --server-port 5500"
+
+$cephnvmf subsystem add --subsystem $NQN --max-namespaces 1024 --no-group-append
```

Comment:
```suggestion
$cephnvmf subsystem add --subsystem $NQN --max-namespaces 1024
```

see [CI](https://github.com/ceph/ceph-nvmeof/actions/runs/13763443952/job/38484937555?pr=1153):
```
++ docker compose run --rm nvmeof-cli --server-address 192.168.13.3 --server-port 5500 subsystem add --subsystem nqn.2016-06.io.spdk:cnode1QOS --max-namespaces 1024 --no-group-append
error: unrecognized arguments: --no-group-append
usage: python3 -m control.cli [-h] [--format {text,json,yaml,plain,python}]
                              [--output {log,stdio}]
                              [--log-level {debug,DEBUG,info,INFO,warning,WARNING,error,ERROR,critical,CRITICAL}]
                              [--server-address SERVER_ADDRESS]
                              [--server-port SERVER_PORT]
                              [--client-key CLIENT_KEY]
                              [--client-cert CLIENT_CERT]
                              [--server-cert SERVER_CERT] [--verbose]
                              {version,gw,spdk_log_level,subsystem,listener,host,connection,namespace,ns,get_subsystems}
```